R/Model-validity.R

```diff
@@ -72,3 +72,16 @@ validate_logistic_log_normal <- function(object) {
   )
   o$result()
 }
+
+#' @describeIn validate_model_objects validates that the model parameters
+#'  are valid as well as `refDose` is a positive scalar.
+validate_logistic_normal <- function(object) {
+  result_lln <- result_validate_logistic_log_normal(object)
```

Comment:
hm... I think either
1) have a helper function for this
2) or call the other one directly (`validate_logistic_log_normal`)

Reply:
Sure, this is a typo here, it should be: `result_lln <- validate_logistic_log_normal(object)`